lib/src/attribute_editor.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:async';
+import 'package:flutter/services.dart';
+
+class AttributeEditor {
+  static const ATTRIBUTE_OPERATION_KEY = "key";
+  static const ATTRIBUTE_OPERATION_TYPE = "action";
+  static const ATTRIBUTE_OPERATION_REMOVE = "remove";
+  static const ATTRIBUTE_OPERATION_SET = "set";
+  static const ATTRIBUTE_OPERATION_VALUE = "value";
+
+  final MethodChannel channel;
+  final String type;
+  final List<Map<String, dynamic>> operations;
+
+  AttributeEditor(String type, MethodChannel channel)
+      : this.type = type,
```

Comment:
Is this colon in the right place? Looks weird.

Reply:
it's in the right place